default_handler.go

```diff
@@ -264,50 +260,45 @@ func (sh *defaultSignalHandler) DeliverSignal(intf, name string, signal *Signal)
 			return
 		default:
 			go func(ch chan<- *Signal) {
+				sh.wg.Add(1)
```

Comment:
If I remember correctly, the convention is to add to a wait group before spawning the goroutine to avoid races between goroutine startup and wg.Wait().

Reply:
My bad, you're right, fixed it.